batch/aiml-workloads/src/cloudbuild.yaml

```diff
@@ -22,8 +22,11 @@ steps:
     - 'build'
     - '-t'
     - 'us-docker.pkg.dev/google-samples/containers/gke/batch-ml-workload'
+    - '-t'
+    - 'us-docker.pkg.dev/google-samples/containers/gke/batch-ml-workload:sample-public-image-${SHORT_SHA}'
     - '.'
   dir: 'batch/aiml-workloads/src'
 
 images:
   - 'us-docker.pkg.dev/google-samples/containers/gke/batch-ml-workload'
+  - 'us-docker.pkg.dev/google-samples/containers/gke/batch-ml-workload:sample-public-image-${SHORT_SHA}'
```

Comment:
`$SHORT_SHA` = the first seven characters of COMMIT_SHA
See [Cloud Build docs here](https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values#using_default_substitutions).

Reply:
awesome that works :D